src/ucp/stream/stream_recv.c

```diff
@@ -231,7 +232,7 @@ ucp_stream_process_rdesc(ucp_recv_desc_t *rdesc, ucp_ep_ext_proto_t *ep_ext,
 static UCS_F_ALWAYS_INLINE void
 ucp_stream_recv_request_init(ucp_request_t *req, ucp_ep_h ep, void *buffer,
                              size_t count, size_t length,
-                             ucp_datatype_t datatype,
+                             ucp_datatype_t datatype, ucs_memory_type_t memory_type,
```

Comment:
`memory_type` is already in `param`, no need to pass it additionaly

Reply:
wanted to check the flag (UCP_OP_ATTR_FIELD_MEMORY_TYPE) only once in the caller for both `ucp_stream_recv_request_init` and `ucp_stream_process_rdesc_inplace`